pyproject.toml

```diff
@@ -67,6 +67,10 @@ test = [
 docker = [
   "uwsgi==2.0.27",
 ]
+sync = [
+  "channels==4.1.0",
+  "daphne==4.1.2",
```

Comment:
Good idea to add a `sync` target here! I think we could also add a few other dependencies in here, such as `websockets` and `pydantic`.

I wonder how this will work with the django management commands, though.

Reply:
> Not sure what you wonder here ? You mean imports that could fail because the package are not installed but still imported automatically ?

That's exactly what I have in mind. We should ensure that these doesn't make the command fail when the `sync` group is not installed.

Reply:
I was thinking about just checking manually, but doing so in the CI wouldn't hurt. It's not a requirement though.

Reply:
Not sure how to do this through the CI, because this would mean changing the deps during the process, or having another dedicated recipe, may be overkill, no ? Or do you have another idea ?